resources/views/queue-monitor/partials/job_action.blade.php

```diff
@@ -0,0 +1,14 @@
+@if($job['failed'])
+<div class="btn-group action-group">
+    <button type="button" class="btn btn-warning">Action</button>
+    <button type="button" class="btn btn-warning dropdown-toggle dropdown-icon" data-toggle="dropdown" aria-expanded="false">
+        <span class="sr-only">Toggle Dropdown</span>
+        <div class="dropdown-menu" role="menu" x-placement="bottom-start" style="position: absolute; transform: translate3d(-1px, 37px, 0px); top: 0px; left: 0px; will-change: transform;">
```

Comment:
Perhaps use a class for it in stylesheet instead of inline styles.

Reply:
Removed inline styling.